src/navigations/root-stack.js

```diff
@@ -95,31 +95,11 @@ export const RootNavigator = ({currentScreen}) => {
     };
   }, []);
 
-  const isUnauthenticated = initialStartup.id === null || initialStartup.id === '';
-
-  const getPaddingTop = (screenName) => {
-    'worklet';
-
-    if (isUnauthenticated || ['TopicPageScreen', 'TopicMemberScreen'].includes(screenName)) {
-      return 0;
-    }
-    return insets.top;
-  };
-
-  const getInsetTopColor = () => {
-    'worklet';
-
-    return currentScreen === 'SampleChatScreen' ? colors.anon_primary : colors.white;
-  };
-
   return (
     <LoadingStartupContext.Provider value={loadingStartup.loadingUser}>
       <View
         style={{
-          height: '100%',
-          paddingBottom: isUnauthenticated ? 0 : insets.bottom,
-          paddingTop: getPaddingTop(currentScreen),
-          backgroundColor: getInsetTopColor()
+          height: '100%'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code removed the dynamic padding and background color based on the current screen. If this was intentional and you've confirmed that it doesn't affect the layout or appearance of your screens, then it's fine. However, if these were removed by mistake, you might want to add them back.
```diff
-        <View
-          style={{
-           height: '100%'
-         }}>
+        <View
+          style={{
+            height: '100%',
+            paddingBottom: isUnauthenticated ? 0 : insets.bottom,
+            paddingTop: getPaddingTop(currentScreen),
+            backgroundColor: getInsetTopColor()
+          }}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->